.cirrus.yml

```diff
@@ -57,6 +50,12 @@ task:
     - mvn --version
     - xsltproc --version
     - xml --version
-  test_script:
+  test_sevntu_script:
+    - cd ci
+    - .ci/validation.cmd sevntu
+  test_verify_without_checkstyle_script:
```

Comment:
Do we need `cd` ?
Can we avoid this ? It is better to run from root of repo

Reply:
> Can we avoid this

Unfortunately no. Caches should be in the working directory. So the layout is
```
working_dir
  |--maven_cache
  |--chocolatey cache
  `--repository_root
```
Another option is
```
working_dir (same as repository_root)
  |--maven_cache
  |--chocolatey_cache
  |--.ci
  |--config
  |--src
  `--so on
```
It will be more safe to separate all directories .